File: Java_Folder/MyLinkedList2.java
public class MyLinkedList2 {
	 
	int number;
	MyLinkedList2 left;
	MyLinkedList2 right;
	
	public MyLinkedList2(int number) {
		this.number=number;
		left=null;
		right=null;
	}
	
	public void setNumber(int number) {
		this.number=number;
	}
	
	public void setLeft(MyLinkedList2 left) {
		this.left=left;
	}
	
	public void setRight(MyLinkedList2 right) {
		this.right=right;
	}
	
	public int getNumber() {
		return(number);
	}
	
	public MyLinkedList2 getLeft() {
		return(left);		
	}
	
	public MyLinkedList2 getRight() {
		return(right);		
	}
	
}

File: Cpp_Folder/MyLinkedList2.cpp
#include "MyLinkedList2.h"

MyLinkedList2::MyLinkedList2(int number)
{
    this->number=number;
    left=nullptr;
    right=nullptr;
}

MyLinkedList2::~MyLinkedList2()
{
    //dtor
}

void MyLinkedList2::setNumber(int number)
{
    this->number=number;
}

void MyLinkedList2::setLeft(MyLinkedList2* left)
{
    this->left=left;
}

void MyLinkedList2::setRight(MyLinkedList2* right)
{
    this->right=right;
}

int MyLinkedList2::getNumber()
{
    return(this->number);
}

MyLinkedList2* MyLinkedList2::getLeft()
{
    return(this->left);
}

MyLinkedList2* MyLinkedList2::getRight()
{
    return(this->right);
}
